Add aten::_upsample_bilinear2d_aa.out

mergennachin opened the pull request:
Summary:

Trying to resolve https://github.com/pytorch/executorch/issues/7031

Vibe-coded using the existing non-alias version in ET and Aten implementation in pytorch as reference, along with reference unittests in pytorch core

Test Plan: 

1. Run https://gist.github.com/mergennachin/9b02aee4feb5acc83e71d8f902f5cca1
And then call `./cmake-out/executor_runner minicpmv_preprocessor.pte`

2. https://gist.github.com/mergennachin/a24e4509804de99caf906c9b79ea45fc

manuelcandales reviewed (approved):
Thank you!

kimishpatel commented:
> along with reference unittests in pytorch core

What does this mean?

kimishpatel commented:
> > along with reference unittests in pytorch core
> 
> What does this mean?

nevermind

mergennachin committed:
Add aten::_upsample_bilinear2d_aa.out (#13458)

Summary:
Trying to resolve https://github.com/pytorch/executorch/issues/7031

Vibe-coded using the existing non-alias version in ET and Aten implementation in pytorch as reference, along with reference unittests in pytorch core

Test Plan:
1. Run https://gist.github.com/mergennachin/9b02aee4feb5acc83e71d8f902f5cca1
And then call `./cmake-out/executor_runner minicpmv_preprocessor.pte`

2. https://gist.github.com/mergennachin/a24e4509804de99caf906c9b79ea45fc

Reviewed By: manuelcandales

Differential Revision: D80343748

Pulled By: mergennachin
